src/types/StateType.ts

```diff
@@ -0,0 +1,36 @@
+import { Socket } from "socket.io-client";
+import { DefaultEventsMap } from "socket.io-client/build/typed-events";
+import { ChatType, UserType, HostType, QuestionType } from "./index";
+
+type StateType = {
+    session: {
+        id: string;
+        name: string;
+        host: HostType;
+        users: UserType[];
+        chats: ChatType[];
+        questions: QuestionType[];
```

Comment:
define it into session type and use sessiontype here
```suggestion
       type SessionType={
       id: string;
        name: string;
        host: HostType;
        users: UserType[];
        chats: ChatType[];
        questions: QuestionType[];
        }
        
        session: SessionType
```